runtime/ts/plan/planificator.ts

```diff
@@ -20,19 +20,18 @@ import {KeyBase} from "../storage/key-base.js";
 import {StorageProviderBase} from "../storage/storage-provider-base.js";
 import {Type} from '../type.js';
 
-type PlanificatorOptions = {
+export type PlanificatorOptions = {
   userid: string;
   storageKeyBase?: string;
   debug?: boolean;
   onlyConsumer?: boolean;
 };
 
-
 export class Planificator {
   static async create(arc: Arc, {userid, storageKeyBase, onlyConsumer, debug = false}: PlanificatorOptions) {
     debug = debug || (storageKeyBase && storageKeyBase.startsWith('volatile'));
-    const store = await Planificator._initSuggestStore(arc, {userid, storageKeyBase, arcKey: null});
-    const searchStore = await Planificator._initSearchStore(arc, {userid, storageKeyBase});
+    const store = await Planificator._initSuggestStore(arc, {userid, storageKeyBase});
+    const searchStore = await Planificator._initSearchStore(arc, {userid, storageKeyBase: null});
```

Comment:
storageKeyBase null is intentional?

Reply:
yes, atm searchStore should always use the same storageKeyBase as the current arc (but i figured it's better to keep the parameter in case we want to change it in the near future).